<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">.QLabel{
color:white;
}
</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <spacer name="verticalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>30</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image:url(:/images/userName.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QFrame" name="frame">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>230</width>
                   <height>65</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
border-image:url(:/images/lineEdit.png);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="userName">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>2000</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color:rgba(0,0,0,0);
border:none;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <spacer name="verticalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>30</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image:url(:/images/password.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>230</width>
                   <height>65</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
border-image:url(:/images/lineEdit.png);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLineEdit" name="passWord">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>2000</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color:rgba(0,0,0,0);
border:none;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="remberPW">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:checked{
	border-image:url(:/images/remPs_checked.png);
}
QPushButton:!checked{
	border-image:url(:/images/remPs_unchecked.png);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="lgBtn">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image:url(:/images/startGame.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="goRegBtn">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image:url(:/images/reg.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>82</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>80</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true">QLineEdit{
border: 3px solid rgb(0, 0, 0);  
border-radius:0px;
color:black; 
};</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>密码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="regUserName"/>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="regPassword"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>用户名</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>手机号</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="regPhonenumber"/>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QPushButton" name="regBtn">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>20</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
border: 5px solid rgb(255, 255, 255);  
border-radius:0px;
background-color:black;
color:white; 
}</string>
               </property>
               <property name="text">
                <string>注册</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>80</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>端口号</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="port">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 3px solid rgb(0, 0, 0);  
border-radius:0px;
color:black; 
};</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="ipAddress">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 3px solid rgb(0, 0, 0);  
border-radius:0px;
color:black; 
};
</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>IP地址</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QPushButton" name="setBtn">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>22</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
border: 5px solid rgb(255, 255, 255);  
border-radius:0px;
background-color:black;
color:white; 
}</string>
               </property>
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="homePageBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/images/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="ipConfiBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/images/setting.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="startSingelMode">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 5px solid rgb(255, 255, 255);  
border-radius:0px;
background-color:black;
color:white; 
}</string>
        </property>
        <property name="text">
         <string>直接开始单机模式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
